index.js

```diff
@@ -23,17 +23,19 @@ function throwError (customLogger, message, error) {
   }
 }
 
-function generateCommands (filePaths, customLogger) {
+function generateCommands(filePaths, customLogger) {
```

Comment:
```suggestion
function generateCommands (filePaths, customLogger) {
```

Reply:
Oh, I've never thought about this before. I think it's a very desirable method!